src/OpenApi/src/Services/OpenApiDocumentService.cs

```diff
@@ -268,6 +268,15 @@ private async Task<Dictionary<OperationType, OpenApiOperation>> GetOperationsAsy
                 var transformer = operationTransformers[i];
                 await transformer.TransformAsync(operation, operationContext, cancellationToken);
             }
+
+            // Apply any endpoint-specific operation transformers registered via
+            // the AddOpenApiOperationTransformer extension method.
+            var endpointOperationTransformers = description.ActionDescriptor.EndpointMetadata
```

Comment:
Should we just check for any `IOpenApiOperationTransformer` metadata?

Reply:
I'm inclined to only check for the delegate-based transformer type that we insert via the overload. We _could_ be more permissive here and support any type `IOpenApiOperationTransformer` that's been set by the user but then we'd have to do the same lifetime management we do for instanced-based transformers (see [here](https://source.dot.net/#Microsoft.AspNetCore.OpenApi/Services/OpenApiDocumentService.cs,143)). 

Also, for that case we might want to consider if we want to do first-class APIs for those types (`AddOpenApiOperationTransformer(IOpenApiOperationTransformer)`) instead of just the delegate overload.